spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  let(:user) do
+    User.create(
+      name: 'Ahmed',
+      photo: 'https://www.google.com/imghp?hl=en&ogbl',
+      bio: 'Full Stack Developer'
+    )
+  end
+
```

Comment:
- [ ] Great work so far but it would be nice if you fix the failing tests which are mentioned by the previous reviewer because all of them are still failing 😨 Kindly fix this let section in both files users and posts spec files to make your tests pass 👍

Reply:
- Updated `schema` to reflect changes to the `users` table resolving bug.
- Updated tests to factor-in changes to `schema`.